### ISSUE TITLE
Add a blocklist of games where shrinking failed

### ISSUE THREAD

**Issue by Louhike** [Louhike/Ugly-GDEmu-Games-Manager#39]
Gditools seems to fail while trying to shrink some games (it either failed immediatly o run eternally).
It will be better if we outright say to the user the game does not work if it has been tested already.

We should add a blacklist and display a warning if the user try to shrink it.

**Comment by Xerxes3rd** [comment]
It appears that megavolt85 knows how to properly shrink WinCE titles, and relayed that info to FamilyGuy (see here: https://www.obscuregamers.com/threads/gdemu-info.413/page-5).  Until FamilyGuy is able to fix gditools, you are welcome to use the list I have here as a starting point for the blacklist: https://docs.google.com/spreadsheets/d/1Mw0AGj2he8tgE8XRojBYy0mspCrgC7uKBrIfCpXD5w8/edit#gid=0

I've gone through every GDI in the latest US TOSEC list (all 298 of them) and tested each one, then updated this list according to my results.  If you export the tab "Problematic Titles" as a CSV, that should work.  I've had success by using the Product ID and Version as a key for a blacklist.  Obviously this won't work for weird hacks that have IP.BIN swaps, but it might be better than nothing?

**Comment by Louhike** [comment]
Yes it's clearly a godd basis and will be quite useful until GdiTools is updated. Thanks!
